extensions/objects/templates/literally-canvas-diagram-editor.component.ts

```diff
@@ -41,6 +62,17 @@ angular.module('oppia').component('literallyCanvasDiagramEditor', {
       ctrl.diagramHeight = 350;
       ctrl.currentDiagramHeight = 350;
       ctrl.data = {};
+      ctrl.diagramStatus = 'editing';
```

Comment:
You're reusing the constant, so make it a named constant and use that everywhere.

Reply:
Done.